spec/predicates/human-review/human-review.md

```diff
@@ -0,0 +1,106 @@
+results of code reviews, software audits, and dependency reviews such as the
+crev project.
+
+This predicate type, meant to be extended by specific systems that emit review
+results. However, those extensions will have a distinct `predicateType` value
+that builds off of the URI in this document.
+
+## Use Cases
+
+Software supply chains encompass many types of human reviews. Best practices
+including standards like SLSA recommend two or multi-party review for source
+code. Another example is the crev project where open source software
+dependencies are socially reviewed.
+
+## Prerequisites
+
+Understanding of the
+[in-toto attestation specification](https://github.com/in-toto/attestation).
+
+## Model
+
+This predicate type includes one compulsory field, `result`, that describes the
+result of the review. Derivatives of this predicate may include other contextual
+fields pertaining to different code review systems or use cases. The predicate
+includes an optional `reviewLink` field that can be used to find the full
+review. In the general case, the subject of a code review attestation must
+identify the software artifacts the review applies to.
+
+## Schema
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [{...}],
+    "predicateType": "https://in-toto.io/attestation/human-review/v0.1",
+    "predicate": {
+        "result": "pass/fail",
+        "reviewLink": "<URL>",
+        "timestamp": "<TIMESTAMP>"
+    }
+}
+```
+
+### Parsing Rules
+
+This predicate follows the in-toto attestation parsing rules. Summary:
+
+-   Consumers MUST ignore unrecognized fields.
+-   The `predicateType` URI includes the major version number and will always
+    change whenever there is a backwards incompatible change.
+-   Minor version changes are always backwards compatible and “monotonic.” Such
+    changes do not update the `predicateType`.
+-   Producers MAY add extension fields using field names that are unlikely to
+    collide with names used by other producers. Field names SHOULD avoid using
+    characters like `.` and `$`.
+-   Fields marked _optional_ MAY be unset or null, and should be treated
+    equivalently. Both are equivalent to empty for object or array values.
+
+### Fields
+
+`result` _interface_, _required_
+
+TODO: what type should this specify? For extensibility, should we have an
+"interface" type?
+
+Indicates the result of the review.
+
+`reviewLink` _URI_, _optional_
+
+Contains a link to the full review. Useful to point to information that cannot
+be captured in the attestation.
+
+`timestamp` _Timestamp_, _optional_
+
+Indicates time of review creation.
+
+## Example
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [{
+        "digest": {
+            "gitCommit": "b0bd8ecab5607e174fa403e002c74a666e7edd51"
+        }
+    }],
+    "predicateType": "https://in-toto.io/attestation/human-review/v0.1",
+    "predicate": {
```

Comment:
Interesting, I would imagine that in some cases the code review system might wind up creating this attestation.  E.g. GitHub creates this attestation for the reviewer, signed with GitHub keys and that GitHub would be the identified attestor.  That's my thinking because GitHub is the system that's actually providing the UI to do the review and so it is in a position to make these attestations.

If it's the individual doing the review that's making/signing the attestation (e.g. with sigstore) that's different.  Of course then there will be the problem of mapping GitHub identities with Sigstore ones (maybe that's already a feature?).

Either way it's probably worth adding text describing who the various attestors could be?